Sources/Commands/SwiftRunTool.swift

```diff
@@ -128,9 +163,18 @@ public class SwiftRunTool: SwiftTool<RunToolOptions> {
     }
     
     /// Executes the executable at the specified path.
-    private func run(_ excutablePath: AbsolutePath) throws {
-        let pathRelativeToWorkingDirectory = excutablePath.relative(to: originalWorkingDirectory)
-        try exec(path: excutablePath.asString, args: [pathRelativeToWorkingDirectory.asString] + options.arguments)
+    private func run(_ excutablePath: AbsolutePath, arguments: [String]) throws {
+        let pathRelativeToWorkingDirectory = excutablePath.relative(to: currentWorkingDirectory)
```

Comment:
>* Rename the existing `--chdir` argument to `--package-path` and change it to simply define the package to work on instead of as a side effect of changing directories.